index.js

```diff
@@ -190,31 +227,35 @@ function cli(process_args) {
                 '                         [--matchByImage REGEXP] [--matchByName REGEXP]\n' +
                 '                         [--skipByImage REGEXP] [--skipByName REGEXP]\n' +
                 '                         [--server HOSTNAME] [--port PORT]\n' +
+                '                         [--debug]\n' +
                 '                         [--help]');
 
     process.exit(1);
   }
 
-  if (argv.port == unbound) {
-    if (argv.secure) {
-      argv.port = 443;
-    } else {
-      argv.port = 80;
-    }
-  } else {
-      argv.port = parseInt(argv.port);
-      // TODO: support service names
-      if (isNaN(argv.port)) {
-        console.log('port must be a number');
-        process.exit(1);
+  const getPort = () => {
+    if (argv.port == undefined) {
+      if (argv.secure) {
+        return 443;
       }
+
+      return 80;
+    }
+
+    // TODO: support service names
```

Comment:
service names?

Reply:
🤷‍♂ i think that TODO is probably here since the beginning